src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+
+type Props = {
+  error: boolean;
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification',
+        'is-danger',
+        'is-light',
+        'has-text-weight-normal',
```

Comment:
also you can combine all requered classes in one string
```suggestion
        'notification is-danger is-ligh has-text-weight-normal',
```